test/e2e/service.go

```diff
@@ -1010,66 +1011,63 @@ func testLoadBalancerNotReachable(ingress api.LoadBalancerIngress, port int) {
 }
 
 func testReachable(ip string, port int) {
-	var err error
-	var resp *http.Response
-
 	url := fmt.Sprintf("http://%s:%d", ip, port)
 	if ip == "" {
-		Failf("got empty IP for reachability check", url)
+		Failf("Got empty IP for reachability check", url)
```

Comment:
Hmm, it looks like this has been broken for a while, actually. The string doesn't contain an `%s` for the URL. Can you please add something at the end like "using URL %s"?